src/views/players/player/player_toxic.cpp

```diff
@@ -47,10 +47,11 @@ namespace big
 			components::player_command_button<"fakeban">(g_player_service->get_selected(), {});
 
 			static int wanted_level;
-			ImGui::SliderInt("WANTED_LVL"_T.data(), &wanted_level, 0, 5);
+			components::small_text("WANTED_LVL"_T.data());
```

Comment:
`.data()` is unnecessary, components::small_text should have support for `std::string_view`.